tutorial_content/benchmarks/01-what-is-benchmark.md

```diff
@@ -0,0 +1,340 @@
+---
+title: What is a Benchmark?
+description: Understanding the fundamentals of Brain-Score benchmarks
+order: 1
+category: benchmarks
+---
+
+# What is a Benchmark?
+
+A **benchmark** is a standardized scientific test that evaluates how well an artificial neural network model aligns with biological intelligence. At its core, a benchmark:
+
+1. **Reproduces an experiment** on an artificial model using the same stimuli and protocol as the original biological experiment
+2. **Compares model responses** to biological measurements using appropriate metrics
+3. **Normalizes scores** using data ceilings to account for measurement noise and variability
+4. **Returns a score** between 0 and 1, where 1 indicates ceiling-level performance
+
+---
+
+## The Brain-Score Philosophy
+
+Brain-Score operates on the principle that AI systems should be evaluated not just on engineering metrics (accuracy, efficiency) but on their alignment with biological intelligence. This requires:
+
+- **Biological grounding**: All benchmarks must be based on actual neuroscience or psychology experiments
+- **Standardized protocols**: Consistent experimental procedures across models
+- **Statistical rigor**: Proper controls, ceilings, and error estimation
+- **Reproducibility**: Clear data provenance and versioning
+
+---
+
+## Components of a Benchmark
+
+Every benchmark is built from four essential components:
+
+```
+┌──────────────────────────────────────────────────────────────────┐
+│                         BENCHMARK                                │
+├──────────────────────────────────────────────────────────────────┤
+│  1. STIMULUS SET                                                 │
+│     • Collection of experimental stimuli (images, text, etc.)    │
+│     • Metadata about each stimulus                               │
+│                                                                  │
+│  2. DATA ASSEMBLY                                                │
+│     • Biological measurements (neural or behavioral)             │
+│     • Experimental conditions and subject information            │
+│                                                                  │
+│  3. METRIC                                                       │
+│     • Statistical comparison method                              │
+│     • Defines how similarity is quantified                       │
+│                                                                  │
+│  4. CEILING                                                      │
+│     • Maximum expected performance given noise                   │
+│     • Enables score normalization                                │
+└──────────────────────────────────────────────────────────────────┘
+```
+
+---
+
+## The Benchmark Interface
+
+Every benchmark implements the `Benchmark` interface:
```

Comment:
Also like this. Another optional add on here (really just for people who are less-CS heavy and might not know what an interface (especially python ABCs) are:

Every benchmark implements the `Benchmark` interface; this means that every benchmark must inlclude the following methods and definitions: